Application/EdFi.Ods.Common/Metadata/Schemas/Ed-Fi-ODS-API-Composites.xsd

```diff
@@ -65,7 +65,7 @@
       <xs:element name="Property" type="Property" />
       <xs:element name="Collection" type="Collection" />
       <xs:element name="ReferencedResource" type="ReferencedResource" />
-      <xs:element name="LinkedCollection" type="LinkedCollection" />
+      <xs:element name="LinkedCollection" type="Resource" />
```

Comment:
Why are we changing to a resource?

Reply:
Because the _only_ reason for the `LinkedCollection` type was to add support for `useHierarchy` attribute on `LinkedCollection` (which is based on `Resource` type) and that attribute is no longer supported.